### DISCUSSION TITLE
Where in OrcaSlicer UI do I turn skirt on or off?

### DISCUSSION THREAD

**Discussion by DougJoseph** [OrcaSlicer/OrcaSlicer#4872] [Q&A]
Where in OrcaSlicer UI do I turn skirt on or off?

So sorry if I'm overlooking something simple, but I've searched both in the interface and on the web and cannot locate it. 

Thanks in advance for any help!

**Comment by DougJoseph** [comment]
OK, so I was advised of the answer on the OrcaSlicer Discord help channel:

I was in the "Objects" mode instead of "Global" mode...

If I switch over to "Global" mode then I see the Skirt settings...

:-)

**Comment by Myshadowslullaby** [comment]
I just wanted to update this because I had trouble too. 
The "Skirt" and the "Brim" settings are now under "Others"
So click Global and under where you see 
Quality | Strength | Speed | Support | Multimaterial | Others
Like me you may have to widen this menu to see it on smaller screens.
![Screenshot 2025-05-28 094602](https://github.com/user-attachments/assets/eb8f6469-d9c0-44c1-9d1a-cfe7d4dafff4)
